procedures/automation-android-setup.md

```diff
@@ -0,0 +1,200 @@
+    2. [Disable the creation of the task that pushes to Google Play](https://github.com/mozilla-mobile/fenix/pull/156/files#diff-73e51d972c105de5122ec559909980daR123)
+    3. Create the PR
+    4. Once approved, merge the PR
+16. Verify the apk artifact(s) of the signing task
+    1. Trigger the non-staging hook
+    2. Once the build finishes, download the apks from the signing task
+    3. Using the prod certificate from step 10.4.1., create a temporary keystore: `keytool -import -noprompt -keystore tmp_keystore -storepass 12345678 -file $product_release.pem -alias $product-rel`
+    4. For each apk, verify that it matches the certificate: `jarsigner -verify $apk -verbose -strict -keystore tmp_keystore`. Check that
+        * The "Digest algorithm" matches step 1
+        * The "Signature algorithm" matches step 1
+        * There are no warnings that there are entries "whose certificate chain invalid", "that are not signed by alias in this keystore" or "whose signer certificate is self-signed"
+    * Do the same thing for the dep signing task and certificate and check that the `jarsigner` command shows that the "Signed by" `CN` is "Throwaway Key"
+17. Request both the creation of a Google Play product and for the credentials to publish to it. Consult with the product team to [fill out the requirements for adding an app to Google Play](https://wiki.mozilla.org/Release_Management/Adding_a_new_app_on_Google_play). This request should be a bug for "Release Engineering > Release Automation: Pushapk", and should be a combination of [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1508294) and [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1512173)
+    * As part of the bug, note that you'll directly send an APK to Sylvestre Ledru and Julien Cristau via Slack
+18. Give the first signed APK to the Google Play admins
+    1. Perform a non-staging build
+    2. Once the signing task is done, grab the APK with the version code of 1 (if there's multiple APKs, you probably want the arm one)
+        * You can verify the version code of the apk with [apktool](https://ibotpeaches.github.io/Apktool/), then viewing the extracted `AndroidManifest.xml` and looking at the `platformBuildVersionCode`
+    3. Send the APK to Sylvestre and Julien
+19. Once the previous step is done and they've set up a Google Play product, put the associated secrets in Hiera
+    1. Connect to VPN and SSH into the puppet master
+    2. Encrypt the `service_account` (you'll have been emailed or slack'd a google service account: it looks like an email address that ends in `gserviceaccount.com`)
+        * `sudo eyaml encrypt --pkcs7-private-key /etc/hiera/keys/private_key.pem --pkcs7-public-key /etc/hiera/keys/public_key.pem --output examples -p -l 'service_account'`
```

Comment:
Nit: Maybe worth adding a pointer to the [official docs](https://wiki.mozilla.org/ReleaseEngineering/PuppetAgain/Secrets#Using_EYAML) on this for more tips?

Reply:
I'll add a link to the encryption docs here, too, good call